### DISCUSSION TITLE
Allow minimize to tray thru tray icon /close/minimize.

### DISCUSSION THREAD

**Discussion by ShiroUwUs7** [DamianMorozov/OpenTgResearcher#38] [Ideas]
Allow minimize to tray thru tray icon /close/minimize. 

and allow option for CLI as throwing it in a terminal tab and letting it run was a lot cleaner for me at least!

**Comment by DamianMorozov** [comment]
Can you explain last option for CLI?

**Reply by ShiroUwUs7** [reply]
![TgDownloaderConsole 1](https://github.com/DamianMorozov/TgDownloader/assets/53096166/726b1885-923d-45b1-af9d-5ea0777beb19)
Simmaler to how this was. where its a few key imputs to do auto download etc. and still in a terminal / cli view. so it will bind to windows terminal as a tab: 

![image](https://github.com/DamianMorozov/TgDownloader/assets/53096166/085f11ff-ab3d-478e-ac00-a01383326a15)

**Reply by ShiroUwUs7** [reply]
also it seems like the sources keep breaking as it won't fetch new ID's no matter how its loaded.

**Comment by ShiroUwUs7** [comment]
Like how it was, the exe opened in command prompt/terminal and was easily
available and minamizable to tray via windows terminal. With the
information etc about the channels, last ID, etc.

**Comment by DamianMorozov** [comment]
For `Windows Terminal`, you can add a new `TgDownloader-X` profile for each instance in a different directory.
![image](https://github.com/DamianMorozov/TgDownloader/assets/13628284/a8970f1a-91ed-403f-94b9-d7729ec79305)

**Reply by DamianMorozov** [reply]
Maybe you mean something like this?
![image](https://github.com/DamianMorozov/TgDownloader/assets/13628284/245dc31f-1f76-4fd3-b70e-e9cb94e4295f)
![image](https://github.com/DamianMorozov/TgDownloader/assets/13628284/08526fc5-918c-4247-893b-aec3808dd9d6)

**Comment by ShiroUwUs7** [comment]
I mean I know how to do the terminal lay out. just the updated one doesn't support it anymore from what I saw.  As its stuck in a gui format.
